contracts/types/GasEstimationTypes.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title GasEstimationType
+ * @notice This enum represents the gas estimation types for different chains.
+ */
+enum GasEstimationType {
+    Default,
+    OptimismBedrock,
+    OptimismEcotone,
```

Comment:
```suggestion
    OptimismEcotone,
    OptimismBedrock,
```
otherwise you'll need to change the values in the configs (and technically the existing state becomes invalid)

Reply:
Yeah it's a breaking change and configs and scripts will need to be updated anyway. Because we now also have an extra scalar value

Reply:
yeah, but one less change needed. Bedrock is the outdated version anyways, will be dropped from most L2s soon. So, fine to keep it lower.